war/src/main/js/config-tabbar.js

```diff
@@ -94,3 +99,22 @@ function addFinderToggle(configTableMetadata) {
 function fireBottomStickerAdjustEvent() {
     Event.fire(window, 'jenkins:bottom-sticker-adjust'); // jshint ignore:line
 }
+// YUI Drag widget does not like to work on elements with a relative position.
+// This tells the element to switch to static position at the start of the drag, so it can work.
+function fixDragEvent(handle){
+    var isReady = false;
+    var $handle = $(handle);
+    var $chunk = $handle.closest('.repeated-chunk');
+    $handle.add('#ygddfdiv')
```

Comment:
I need to catch and control the position style of the element both before it is dragged and while it is being dragged. When it is being dragged, YUI tosses the element into '#ygddfdiv'. If I had another way to track the element, I would prefer it, but don't want to dive into YUI to figure it out. The name of the ID seems to imply that this is the div all drag elements will get tossed into.